release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -528,7 +523,12 @@ In the following tables, features are marked with the following statuses:
 [id="ocp-4-14-nodes-cgroupv1-deprecated"]
 ==== Linux Control Groups version 1 is now deprecated
 
-In {op-system-base-full} 9, the default mode is cgroup v2. When {op-system-base-full} 10 is released, systemd will not support booting in the cgroup v1 mode and only cgroup v2 mode will be available. As such, cgroup v1 is deprecated in {product-title} 4.16 and later. cgroup v1 will be removed in a future {product-title} release. 
+In {op-system-base-full} 9, the default mode is cgroup v2. When {op-system-base-full} 10 is released, systemd will not support booting in the cgroup v1 mode and only cgroup v2 mode will be available. As such, cgroup v1 is deprecated in {product-title} 4.16 and later. cgroup v1 will be removed in a future {product-title} release.
```

Comment:
**Note to reviewer:** This content is coming in as I rebased conflicts for 4.16 and accepted the changes made to the RN. So this content is already reviewed and merged.